src/components/organisms/WithInformationCard/WithInformationCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import { AnyVenue } from "types/venues";
+
+import { InformationLeftColumn } from "components/organisms/InformationLeftColumn";
+import InformationCard from "components/molecules/InformationCard";
+
+import "./WithInformationCard.scss";
+
+interface PropsType {
+  venue: AnyVenue;
+  showInfo?: boolean;
+}
+
+export const WithInformationCard: React.FunctionComponent<PropsType> = ({
+  venue,
+  showInfo = false,
+}) => (
+  <>
+    {showInfo && (
+      <InformationLeftColumn iconNameOrPath={venue?.host?.icon || "info"}>
+        <InformationCard>
+          <p className="information-title">{venue.name}</p>
```

Comment:
Why are we adding the title here separately, after making it optional in `InformationCard`?

Why not just use the `title` prop on `InformationCard` as intended?

```suggestion
        <InformationCard title={venue.name}>
```